Malformed package.json

Ran commands to run ionic as directed in the readme but the program failed to run
```
$ ionic serve
[ERROR] The package.json file seems malformed.
```
The [base requirements](https://docs.npmjs.com/getting-started/using-a-package.json#requirements) include `name` and `version` which are both missing from the ionic [`package.json`](https://github.com/FMCorz/react-native-vs-ionic-todos/commit/2db34a76629f0a4676ac838fcc94ab961c2027ec#diff-053d44df59719ccd02d2322c4205a698)

--- a/ionic2/package.json
+++ b/ionic2/package.json
@@ -1,4 +1,6 @@
 {
+  "name": "react-native-vs-ionic-todos",
+  "version": "0.0.1",
   "dependencies": {
     "@angular/common": "2.0.0-rc.4",
     "@angular/compiler": "2.0.0-rc.4",

